### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Mnpn03**
+ I've added a small addition to ping.
+ Changed "[CENSORED]" to "[HIDDEN]" as it looks better IMO, I don't know what you think.
- Changed version number, idk if you're okay with that.

Minor changes, revert if you want!

### CODE REVIEW

File: intercept.go [hunk 1]
```diff
@@ -131,7 +131,8 @@ func messageCommand(session *discordgo.Session, e *discordgo.Message, guild *dis
 		}
 
 		inMS := timestamp.Sub(first).Nanoseconds() / time.Millisecond.Nanoseconds()
-		text := "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
+		text += "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
```
Comment (LEGOlord208) [inline intercept.go]: Pretty sure that won't work....

File: intercept.go [hunk 2]
```diff
@@ -131,7 +131,8 @@ func messageCommand(session *discordgo.Session, e *discordgo.Message, guild *dis
...
+		text += "Calculating outgoing.."
```
Comment (LEGOlord208) [inline intercept.go]: Good, but that text is never cleared later, so...

File: main.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const AutoRunFile = ".autorun"
-const Version = "1.22.4dev"
+const Version = "1.22.5dev"
```
Comment (LEGOlord208) [inline main.go]: The version goes by the future released version + dev. So doing this, you just skipped a version.

Comment (Mnpn03) [inline main.go]: Okay, fixing. I also wrote in my post "Changed version number, idk if you're okay with that.".

File: intercept.go [hunk 4]
```diff
@@ -130,9 +130,16 @@ func messageCommand(session *discordgo.Session, e *discordgo.Message, guild *dis
...
-		inMS := timestamp.Sub(first).Nanoseconds() / time.Millisecond.Nanoseconds()
-		text += "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
-		text += "Calculating outgoing.."
+		inMS := first.Sub(timestamp).Nanoseconds() / time.Millisecond.Nanoseconds()
+
+		// Discord 'bug' makes us receive the message before the timestamp, sometimes.
+		text := ""
+		if inMS < 0 {
+			text += "Incoming: `" + strconv.FormatInt(inMS, 10) + "ms`"
+		  text += "Calculating outgoing.."
```
Comment (LEGOlord208) [inline intercept.go]: Still, that text is never cleared later. Also, I don't think you meant for it to be in the if check

**Pull request merged**